src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java

```diff
@@ -130,10 +138,11 @@ OssindexClient buildOssIndexClient() {
 
       if (extension.isSimulatedVulnerabilityFound()) {
         ComponentReportVulnerability vulnerability = new ComponentReportVulnerability();
+        vulnerability.setId("123-456-789");
         vulnerability.setTitle("Simulated");
         vulnerability.setCvssScore(4f);
         vulnerability.setReference(new URI("http://test/123"));
-        report.setVulnerabilities(Arrays.asList(vulnerability));
+        report.setVulnerabilities(Lists.newArrayList(vulnerability));
```

Comment:
Why this changed?

Reply:
> Why this changed?

`Arrays.asList` returns an immutable list so when we try it remove the vulnerabilities it will throw an exception. `Lists.newArrayList` returns a mutable list.